mne/tests/test_proj.py

```diff
@@ -122,6 +122,8 @@ def test_compute_proj_epochs():
                 p2_data = p2_data[:, mask]
             corr = np.corrcoef(p1_data, p2_data)[0, 1]
             assert_array_almost_equal(corr, 1.0, 5)
+            if p2['explained_var']:
+                assert_true(p1['explained_var'] == p2['explained_var'])
```

Comment:
like between 0 and 1?

I guess we could also add tests to our `preprocessing` module for EOG/ECG that the values are at least 0.1 or something.